Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -23,6 +23,60 @@ The web server for the integration runs within a long-running Docker container.
  `...`
  `-----END PRIVATE KEY-----`
 ```
+ - Microsoft does not support self-signed certificates and requires a chain-trusted certificate issued by a trusted CA.
+ 
+## Setup Examples
+
+### 1. Using Cortex XSOAR rerouting
+In this configuration, we will use Cortex XSOAR functionality, which reroutes HTTPS requests that hit the default port (443) to the web server that the integration spins up.
+
+The messaging endpoint needs to be: `<CORTEX-XSOAR-URL>/instance/execute/<INTEGRATION-INSTANCE-NAME>`, e.g. `https://my.demisto.live/instance/execute/teams`
+
+The integration instance name, `teams` in this example, needs to be configured in the [Configure Microsoft Teams on Demisto](#configure-microsoft-teams-on-demisto) step.
+
+The port to be configured in [Configure Microsoft Teams on Demisto](#configure-microsoft-teams-on-demisto) step should be any available port that is not used by another service.
+
+ - Note: This option is available from Cortex XSOAR v5.5.0 and later.
+
+### 2. Using NGINX as reverse proxy
+In this configuration, the inbound connection, from Microsoft Teams to Cortex XSOAR, goes through a reverse proxy (e.g. NGINX) which relays the HTTPS requests posted from Microsoft Teams
+to the Cortex XSOAR server on HTTP.
+
+On NGINX, configure the following:
+ - SSL certificate under `ssl_certificate` and `ssl_certificate_key`
+ - The Cortex XSOAR server (including the port) under `proxy_pass`, e.g. `http://mydemistoinstance.com:7000`
+ 
+Follow [Configuring Upstream Servers NGINX guide](https://docs.nginx.com/nginx/admin-guide/security-controls/securing-http-traffic-upstream/#configuring-upstream-servers) for more details.
+
+The port (`7000` in this example), to which the reverse proxy should forward the traffic on HTTP, should be the same port you specify in the integration instance configuration, as the webserver the integration spins up, listens on that port.
+
+![image](./doc_files/RP-NGINX.png)
+
+![image](./doc_files/InstanceConfig7000.png)
+
+### 3. Using Apache reverse proxy and Cortex XSOAR engine
+In this configuration, the inbound connection, from Microsoft Teams to Cortex XSOAR, goes through a reverse proxy (e.g. Apache) and possibly a load balancer, which relays the HTTPS requests posted from Microsoft Teams
+to a Cortex XSOAR engine, which can be put in a DMZ, on HTTP.
+
+The port (`7000` in this example), to which the reverse proxy should forward the traffic on HTTP, should be the same port you specify in the integration instance configuration, as the webserver the integration spins up, listens on that port.
```

Comment:
Let's also add a link to: https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html